### DISCUSSION TITLE
What's the difference between the bitnami and the bitnami-labs chart?

### DISCUSSION THREAD

**Discussion by rbjorklin** [bitnami-labs/sealed-secrets#1499] [Q&A] [answered]
I've found two charts that look "official" but I'm not sure which one is recommended and why I should prefer one over the other:

* https://github.com/bitnami-labs/sealed-secrets/tree/main/helm/sealed-secrets
* https://github.com/bitnami/charts/tree/main/bitnami/sealed-secrets

Would appreciate it if someone could shed some light on the situation.

**Comment by agarcia-oss** [comment, marked as answer]
Hi @rbjorklin, 

The official Sealed Secrets Helm chart is the one you can find in the repository https://github.com/bitnami-labs/sealed-secrets/tree/main/helm/sealed-secrets
The Sealed Secrets team maintains and supports this Helm chart. 

The Bitnami chart is another option to deploy the Sealed Secrets controller,  but we do not maintain this chart directly. To access the latest chart features, we recommend the use of the official, upstream chart.

Regards
